<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowDrugs</class>
 <widget class="QWidget" name="ShowDrugs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>4med::Просмотр лекарственных средств</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="RussianFederation"/>
  </property>
  <widget class="QListView" name="drugsList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>21</y>
     <width>123</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="bRegistration">
      <property name="text">
       <string>Приход/расход</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bEdit">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bDel">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
